src/NzbDrone.Core/ImportLists/Simkl/User/SimklUserImport.cs

```diff
@@ -17,14 +23,40 @@ public class SimklUserImport : SimklImportBase<SimklUserSettings>
         {
         }
 
-        public override string Name => "Simkl User Watchlist";
+        public override string Name => "Simkl Shows Watchlist";
+
+        public override IImportListRequestGenerator GetRequestGenerator()
+        {
+            return new SimklUserRequestGenerator()
+            {
+                Settings = Settings,
+                ClientId = ClientId,
+                ShowType = SimklShowTypes.Shows
+            };
+        }
+    }
+
+    public class SimklAnimeImport : SimklImportBase<SimklUserSettings>
```

Comment:
A separate class for anime isn't needed, as mentioned this can be done via a setting on `SimklUserSettings` instead.

Reply:
A dropdown for selecting anime or shows instead of the separate button. You should be able to use `ListType` on `SimklUserSettings` as a guide on how it should be implemented.

Reply:
In the case of other lists the lists are quite different, in this case it's just a different type of series, not too different from a different list type as it's all coming from the same source.